src/fpd/enrichment.js

```diff
@@ -98,21 +98,28 @@ function removeUndef(obj) {
   return getDefinedParams(obj, Object.keys(obj))
 }
 
+// export async function tryToGetCdepLabel(cb = getCookieDeprecationLabel) {
+//   let cdep;
+//   if (isActivityAllowed(ACTIVITY_ACCESS_DEVICE, activityParams(MODULE_TYPE_PREBID, 'cdep'))) {
+//     return GreedyPromise.resolve(
+//       await cb(cdep)
+//     );
+//   }
+// }
+
 export async function tryToGetCdepLabel(cb = getCookieDeprecationLabel) {
-  let cdep;
-  if (isActivityAllowed(ACTIVITY_ACCESS_DEVICE, activityParams(MODULE_TYPE_PREBID, 'cdep'))) {
-    return GreedyPromise.resolve(
-      await cb(cdep)
-    );
-  }
+  return GreedyPromise.resolve(isActivityAllowed(ACTIVITY_ACCESS_DEVICE, activityParams(MODULE_TYPE_PREBID, 'cdep')) && await cb());
 }
 
-function getCookieDeprecationLabel(cdl) {
+function getCookieDeprecationLabel() {
   return new Promise((resolve) => {
     navigator.cookieDeprecationLabel.getValue().then((label) => {
-      if (label) cdl = label;
-      resolve(cdl);
+      label = label || 'example_label';
+      resolve(label);
     });
+    setTimeout(() => {
```

Comment:
we have exising examples of FPD that waits for promises without timeouts (like user agent hints). I think it's OK because none of them actually need to wait for anything - there's no network calls or user prompts as far as I can tell, they should always resolve (or reject) very quickly. If we do introduce a timeout it should be configurable, and this should reject - not resolve to false.

Reply:
Ok, thanks for the input!  Sounds like we are ok without the setTimeout in this case at this time then.  I removed it for now.